src/diffusers/schedulers/scheduling_dpmsolver_multistep.py

```diff
@@ -323,6 +335,12 @@ def _sigma_to_t(self, sigma, log_sigmas):
         t = t.reshape(sigma.shape)
         return t
 
+    def _sigma_to_alpha_sigma_t(self, sigma):
```

Comment:
actually, originated here and not copied from anywhere 😛

Reply:
hi @yiyixuxu, question from a casual observer here, following along and converting UniPC into jax... it seems the `_sigma_to_alpha_sigma_t` is used up to 9X in every scheduler step (once in `convert_model_output`, twice in s0 and t versions in each the C and P multistep update functions, and up to twice again in each according to the `order` determination of that step, which would most commonly be 2 or 3, so 1X or 2X more again)

is this function used elsewhere in other schedulers in a way that makes it impossible to run the calculation once and set all `sigmas` and `alphas` within the `set_timesteps` function? from what i can tell during my tinkering, `alphas`, `lambdas`, `sigmas` (and scheduler-specific things like `R`, `rks`, `rhos`, `order`, etc.) are only dependent on `timestep` and input parameters, not any data/sample values during the `step` process, so they can be created in advance and then referenced from the `state` (or `self` for torch) variable using `[step_index]` instead of `[timestep]`, which all could maybe save some compute (and lines of code?)

Reply:
hey! sorry I missed this comment. a good suggestion! feel free to open an PR, otherwise I will find a time to work on this soon